src/osd/PrimaryLogPG.cc

```diff
@@ -3824,6 +3824,12 @@ void PrimaryLogPG::do_scan(
       } else {
 	// we canceled backfill for a while due to a too full, and this
 	// is an extra response from a non-too-full peer
+        ostringstream ss;
+        if (osd->check_backfill_full(ss)) {
+          dout(20) << __func__
+            << ": Canceling backfill for a while due to a too full "
```

Comment:
@shinobu-x "due to a too full" Looks wrong sentence.
Instead it can be "Cancelling backfill. It's full."